finished-Scripts/AvatarScripts/City_of_Walls_and_Secrets.xml

```diff
@@ -646,7 +647,7 @@
    </section>
    <section>
       <speech speaker="Katara">
-         <charAction>Frontal view.</charAction> Excuse me? I've>
+         <charAction>Frontal view.</charAction> Excuse me? I've&gt;
```

Comment:
@julianjmg I just noticed something in the files from doing this PR (thanks to GitHub's "diff" view): There seem to be a lot of `>` characters scattered in the files. I think my computer "flattened" them to `&gt;` in this push (not something I was trying to do). GitHub has sort of highlighted them all in this commit. I think we could try a quick run of "Find and Replace in Files" to clean this up. It will matter when we do the Natural Language Processing next week, because the `>` will likely get in the way of the processing a little.